### ISSUE TITLE
More method to update list

### ISSUE THREAD

**Issue by voliothonh** [liangjingkanji/BRV#411]
I want simple to update list (add/remove/change) by one method like: 
fun updateData(updatedList : List<Any>){

}

- If updateList same originList before => No change
- If any Object index diffirent => update that row (Even some variable of Object diffirent still update list)

**Comment by liangjingkanji** [comment]
This will result in adding a bunch of methods, More learning costs

I think it's meaningless

**Comment by liangjingkanji** [comment]
```kotlin
rv.models = newList // set new data
rv.addModels(data)  // add data
```

**Comment by voliothonh** [comment]
> This will result in adding a bunch of methods, More learning costs
> I think it's meaningless

I agree with you, that method will add more many of code and costs. But it's very easy to use and meaning.
I take that ideal from epoxy recyclerview. 
When you have time you can consider it further.

Thank for Your library, It's great, improving every day.

**Comment by liangjingkanji** [comment]
`epoxy ` very difficult to use, It is the most expensive rv library to learn in the world
